Assets/MRTK/Core/Providers/InputSimulation/InputSimulationEnum.cs

```diff
@@ -12,6 +12,16 @@ public enum InputSimulationControlMode
         Walk,
     }
 
+    public enum EyeGazeSimulationMode
+    {
+        // Disable eye gaze simulation
```

Comment:
Since this is public, making these `<summary>` tags would be good for docs (and documenting the enum itself)